.github/workflows/test.yml

```diff
@@ -103,12 +103,30 @@ jobs:
         run: |
           gsutil cp ./sauce-playwright-runner.zip gs://${{ secrets.GCS_RUNNER_BUCKET }}/sauce-playwright-runner-${{ github.run_id }}.zip
 
+  windows-bundle-test:
+    needs: [build-windows-bundle]
+    runs-on: windows-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

Comment:
Mind updating the entire workflow to a newer `actions/checkout` and `action/setup-node`? Would be great!